kube-monitoring/charts/templates/tests/test-kube-monitoring-config.yaml

```
@@ -53,6 +53,9 @@ data:
     @test "Verify creation of the prometheus-{{ .Release.Name }} statefulset" {
         verify "there is 1 statefulset named 'prometheus-{{ .Release.Name }}'"
     }
+    @test "Verify creation of the {{ .Release.Name }}-sd Podmonitor" {
```

Adding a blank line to be consistent like other test cases. 
```suggestion

    @test "Verify creation of the {{ .Release.Name }}-sd Podmonitor" {
```